frontend/scss/components/organisms/stage-ads.scss

```diff
@@ -0,0 +1,122 @@
+@import '../../extends';
+@import '../../functions';
+@import '../../mixins';
+@import '../../variables';
+
+@import '../atoms/_color.scss';
+@import '../atoms/_icon.scss';
+@import '../atoms/_headline.scss';
+@import '../atoms/_text.scss';
+
+.#{utility('stage')} {
+  overflow: hidden;
+}
+
+.#{organism('stage')} {
+  @include gradient-ads;
+  grid-column: 1 / -1;
+  position: relative;
+  width: 100%;
+  margin-left: 30px;
+  border-bottom-left-radius: 4em;
+  -webkit-clip-path: polygon(0 0, 100% 0, 100% calc(100% - 2em), 4em 100%, 0% 100%);
+  clip-path: polygon(0 0, 100% 0, 100% calc(100% - 2em), 4em 100%, 0% 100%);
+
+  @media (min-width: 768px) {
+    margin-left: 64px;
+    border-bottom-left-radius: 6.250em;
+    -webkit-clip-path: polygon(0 0, 100% 0, 100% calc(100% - 3em), 6.250em 100%, 0% 100%);
+    clip-path: polygon(0 0, 100% 0, 100% calc(100% - 3em), 6.250em 100%, 0% 100%);
+  }
+
+
+  div.#{utility('container-fluid')} {
```

Comment:
Does that selector need to be that specific by prepending `div`?

Reply:
all stage organisms are wrapped in a <section> tag that utilizes the "ap--container-fluid" class, figured using this selector to style the individual div was less css than creating a whole new utility class with the same properties as "ap--container-fluid" with the exception for a different padding